### DISCUSSION TITLE
Is it possible to change the generated style.css imports order?

### DISCUSSION THREAD

**Discussion by TheOrdinaryDev** [chakra-ui/panda#899] [Q&A] [answered]
With `preflight: true` and `staticCss` config options generated `style.css` file looks like this
```css
@layer reset, base, tokens, recipes, utilities;

@import './global.css';

@import './static.css';

@import './reset.css';

@import './tokens/index.css';

@import './tokens/keyframes.css';
```
`reset.css` is after `static.css`, so resetting styles overrides static styles.

Is it possible to change imports order in generated `style.css` file?

**Comment by astahmer** [comment, marked as answer]
it's currently not possible to change the imports order, I don't see any downsides to having the `reset.css` at the top of the imports
done here https://github.com/chakra-ui/panda/pull/902
